wagtail/urls.py

```diff
@@ -18,7 +18,7 @@
 
 
 WAGTAIL_FRONTEND_LOGIN_TEMPLATE = getattr(
-    settings, "WAGTAIL_FRONTEND_LOGIN_TEMPLATE", "wagtailcore/login.html"
+    settings, "WAGTAIL_PASSWORD_REQUIRED_TEMPLATE", "wagtailcore/login.html"
```

Comment:
This is not the correct line to update - we're renaming `PASSWORD_REQUIRED_TEMPLATE`, not `WAGTAIL_FRONTEND_LOGIN_TEMPLATE`.

Also, we need to keep the old setting names working for a while, so that we're not breaking existing projects - see https://docs.wagtail.org/en/stable/contributing/release_process.html#deprecation-policy. In this case, we should make sure that any project that defines `PASSWORD_REQUIRED_TEMPLATE` / `DOCUMENT_PASSWORD_REQUIRED_TEMPLATE` (rather than the new `WAGTAIL_` settings) continues to work, but raises a `RemovedInWagtail70Warning`. We will then drop support for the old names in Wagtail 7.0.

Reply:
yes , had a bit of confusion on this one , so gonna revert the changes on this file👍

Reply:
\wagtail\wagtail\models\__init__.py found another file with instance ` password_required_template = getattr(
        settings, "PASSWORD_REQUIRED_TEMPLATE", "wagtailcore/password_required.html"
    )` so gonna change it to ` password_required_template = getattr(
        settings, "WAGTAIL_PASSWORD_REQUIRED_TEMPLATE", "wagtailcore/password_required.html"
    )`

Reply:
Do I have to update all the Markdowns files too having these instances ?

Reply:
Yes, the documentation needs to be updated to use the new names. It would also be a good idea to add [a `versionchanged` note](https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#version-added-changed-deprecations) to advise users that this has changed:

    ```{versionchanged} 6.1
    This setting was renamed from `PASSWORD_REQUIRED_TEMPLATE` to `WAGTAIL_PASSWORD_REQUIRED_TEMPLATE`.
    ```

Reply:
hi , not sure as to where to add ```{versionchanged} 6.1
This setting was renamed from `PASSWORD_REQUIRED_TEMPLATE` to `WAGTAIL_PASSWORD_REQUIRED_TEMPLATE`.
``` as found many instances , kindly hyperlink the file where should i !

Reply:
This would be in https://github.com/wagtail/wagtail/blob/main/docs/reference/settings.md, after the description of the `PASSWORD_REQUIRED_TEMPLATE` setting. Here's an example of where we've done this previously: https://github.com/wagtail/wagtail/blob/c5caec63450a4389dd1cfe571b3605a6fc8006c8/docs/reference/settings.md?plain=1#L794-L805

Reply:
kindly review now 👍